doc/cygnus-ngsi/user_and_programmer_guide/unrestricted_type.md

```diff
@@ -0,0 +1 @@
+An Attribute value of a TextUnrestricted type from an ContextBroker event is escaped from character ' to ''
```

Comment:
```suggestion
Cygnus escapes from `'` to `''` in attributes of type `TextUnrestricted` with the aim of avoiding injection attacks.

Note that other attributes (i.e. with type different to `TextUnrestricted` don't need such escaping as single quote (`'`) is a [forbidden character in Context Broker](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html) so that value will never arrives to Cygnus in notifications.
```

To align the text with the one used in the other .md file.